changelog.mdx

```diff
@@ -4,6 +4,11 @@ description:
   "We release features and fixes weekly! See some of the changes we have made."
 ---
 
+## 0.3.0
+
+#### Date: 25 November 2022
+- We've switched out infrastructure provider for something more reliable and faster. This means faster deployment, less downtime and roughly half the inference time!
```

Comment:
I wouldn't say we switched out our infra provider but rather "We made large changes to our infrastructure which means faster deployment etc"

Reply:
++